custom_components/eo_mini/sensor.py

```diff
@@ -87,9 +89,6 @@ def _handle_coordinator_update(self) -> None:
 
         if self.coordinator.data:
             if self.coordinator.data["ESKWH"] == 0:
-                self._attr_last_reset = datetime.fromtimestamp(
```

Comment:
This bug was already fixed by changing the state class of the sensor to be a "total" state class in b2309d117ba859, so this conflicts.

Reply:
The bug you are linking to is a different one. That changes it for `EOMiniChargerSessionEnergySensor` but my fix here is for `EOMiniChargerSessionChargingTimeSensor`

Reply:
For sure they can, I will submit a followup MR so these are consistent